src/app/containers/IndexPageSection/index.jsx

```diff
@@ -166,13 +166,16 @@ const sectionBody = ({
 const IndexPageSection = ({ bar, group, sectionNumber, showAllPromos }) => {
   const { script, service, dir, translations } = useContext(ServiceContext);
   const sectionLabelId = idSanitiser(group.title);
+  const { topStoriesTitle } = translations;
 
-  const strapline = pathOr('', ['strapline', 'name'], group);
   const isLink = pathOr(null, ['strapline', 'type'], group) === 'LINK';
   const href = pathOr(null, ['strapline', 'links', 'mobile'], group);
   const type = pathOr(null, ['type'], group);
   const seeAll = pathOr(null, ['seeAll'], translations);
   const isFirstSection = sectionNumber === 0;
+  const strapline = isFirstSection
+    ? pathOr(topStoriesTitle, ['strapline', 'name'], group)
+    : pathOr('', ['strapline', 'name'], group); // We are not supposed to default this value if the group is not the 1st section
```

Comment:
This has now overwritten the snapshot that checks for an empty string. Maybe we should re-add a snapshot test for this logic.

Reply:
Added 2 snapshot tests for this.